index/upsidedown/index_reader.go

```diff
@@ -15,11 +15,12 @@
 package upsidedown
 
 import (
+	"context"
 	"reflect"
 
 	"github.com/blevesearch/bleve/v2/document"
 	index "github.com/blevesearch/bleve_index_api"
-	"github.com/blevesearch/upsidedown_store_api"
+	store "github.com/blevesearch/upsidedown_store_api"
```

Comment:
This changes seems unrelated?

Reply:
This popped up because of the vscode extension I believe, removing it.